t/sanity.t

```diff
@@ -7,7 +7,7 @@ use Test::Nginx::Socket::Lua;
 #workers(2);
 #log_level('warn');
 
-repeat_each(2);
+repeat_each(1);
```

Comment:
@wangfakang Then you should add a new `.t` file for these tests that cannot run twice and leave existing test cases intact.